assets/models/system/automl-ner/spec.yaml

```diff
@@ -0,0 +1,51 @@
+$schema: https://azuremlschemas.azureedge.net/latest/model.schema.json
+name: AutoML-Named-Entity-Recognition
```

Comment:
nit: maybe AutoML-Text-Named-Entity-Recognition is better, but no strong preference

Reply:
Hi @bhimar , I might have to dismiss this comment since I feel it's more natural to say "ner", rather than "text-ner". But I do see your point that you want to have unified name format for NLP tasks.